.github/in-cluster-test-scripts/external-workloads-install.sh

```diff
@@ -5,11 +5,11 @@ set -e
 
 # Install Cilium in cluster
 cilium install \
-  --cluster-name "${CLUSTER_NAME}" \
-  --config monitor-aggregation=none \
+  --helm-set=cluster.name="${CLUSTER_NAME}" \
+  --helm-set=extraConfig.monitor-aggregation=none \
   --config tunnel=vxlan \
```

Comment:
should we change this as well ? basically, I am fine with either direction (e.g. --helm-set=extraConfig vs --config), just more about consistency in the same file/step.